rudder-agent/SOURCES/debug-script/tests/rudder_unsent_inventories_agent

```diff
@@ -20,11 +20,19 @@ rudder_unsent_inventories_agent() {
   else
     UNSENT_INVENTORIES_DIR="${RUDDER_VAR}/inventories"
   fi
-
-  COMMAND_OUTPUT=`ls -l "${UNSENT_INVENTORIES_DIR}" 2>&1 | wc -l`
+  recent_time="432000"
```

Comment:
Add a comment to tell how much it make sin hours or days

Reply:
added